packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -54,6 +54,16 @@ function registerCommands(): vscode.Disposable {
     'sfdx.force.source.status',
     forceSourceStatus
   );
+  const forceSourceStatusLocalCmd = vscode.commands.registerCommand(
+    'sfdx.force.source.status.local',
+    forceSourceStatus,
+    { flag: '--local' }
```

Comment:
registerCommand takes a third parameter and binds it to the `this` context of forceSourceStatus.